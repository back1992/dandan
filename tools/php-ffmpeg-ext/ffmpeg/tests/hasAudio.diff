001+ Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/hasAudio.php on line 2
001- ffmpeg hasAudio(): No
002- ffmpeg hasAudio(): Yes