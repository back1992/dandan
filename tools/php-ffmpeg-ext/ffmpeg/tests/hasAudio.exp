ffmpeg hasAudio(): No
ffmpeg hasAudio(): Yes