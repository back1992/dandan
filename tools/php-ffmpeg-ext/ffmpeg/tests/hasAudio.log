
---- EXPECTED OUTPUT
ffmpeg hasAudio(): No
ffmpeg hasAudio(): Yes
---- ACTUAL OUTPUT
Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/hasAudio.php on line 2
---- FAILED
