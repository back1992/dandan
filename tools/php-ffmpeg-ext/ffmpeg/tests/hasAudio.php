<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/test.avi');
printf("ffmpeg hasAudio(): %s\n", $mov->hasAudio() ? 'Yes' : 'No');
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/robot.avi');
printf("ffmpeg hasAudio(): %s\n", $mov->hasAudio() ? 'Yes' : 'No');
?>
