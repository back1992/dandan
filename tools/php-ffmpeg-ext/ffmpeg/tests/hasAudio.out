Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/hasAudio.php on line 2